config/sync/language/da/block.block.claro_breadcrumbs.yml

```diff
@@ -1,2 +1,2 @@
 settings:
-  label: Brødkrummer
+  label: Breadcrumbs
```

Comment:
This replaces a Danish text "Brødkrummer" with an English text "Breadcrumbs" in a Danish translation file. That does not seem intentional - nor correct. The same goes for the rest of this PR.

This needs to be investigated further. Perhaps it is related to #849?